go/chat/convmeta.go

```diff
@@ -0,0 +1,25 @@
+package chat
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/keybase/client/go/chat/storage"
+	"github.com/keybase/client/go/protocol/chat1"
+)
+
+func ConversationMetadata(ctx context.Context, ri chat1.RemoteInterface, convID chat1.ConversationID, rl *[]*chat1.RateLimit) (chat1.Conversation, error) {
```

Comment:
I was going to replace this guy with a call into the current `InboxSource`, so it probably doesn't need to be factored out into its own file.